_posts/2017-02-21-ome-files-cpp-0-3-1.md

```diff
@@ -76,7 +76,7 @@ Note that this is a work in progress and subject to change.
 
 **Errata**
 
-See [Errata](https://www.openmicroscopy.org/site/support/ome-files-cpp/ome-cmake-superbuild/manual/html/errata.html) for this release.
+See [Errata](https://docs.openmicroscopy.org/ome-cmake-superbuild/0.3.1/errata.htm) for this release.
```

Comment:
:+1: will push the fix

Good point about linkchecking. We are currently disabling external linkcheck in Travis and limiting it to internal checks. I had a quick investigation at enabling it back but as we see in our daily job builds, it comes with its lot of false positives and we don't want Travis to be systematically red.

The plan for 1.0.0 was minimally for someone to run it manually, curate the output and fix the failing URLs. See https://trello.com/c/6mhivdqz/7-fix-broken-links for more information

What probably needs to happen if fine the logic in `htmlproofer` to exclude `href` or restrict domains (`*.openmicroscopy.org` would be a good start). Suggestions/help welcome.